{"nick":"dbevenius","date":"2019-04-17T00:08:41.129Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 246 seconds","date":"2019-04-17T00:13:15.032Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T00:28:40.362Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 246 seconds","date":"2019-04-17T00:32:58.307Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T00:52:08.005Z","type":"join"}
{"nick":"joaocgreis","message":"https://ci.nodejs.org/computer/test-scaleway-ubuntu1604-armv7l-1/ back online","date":"2019-04-17T00:55:23.573Z","type":"message"}
{"nick":"dbevenius","reason":"Ping timeout: 250 seconds","date":"2019-04-17T00:56:34.782Z","type":"quit"}
{"nick":"rvagg","message":"joaocgreis: hm, I only recently had to bring that one back online with a forced restart, perhaps we're having hardware problems.. let's remember to monitor and if it happens again to this machine we should think about alternative solutions.","date":"2019-04-17T01:06:58.190Z","type":"message"}
{"nick":"dbevenius","date":"2019-04-17T01:08:37.461Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 244 seconds","date":"2019-04-17T01:13:08.350Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T02:08:41.131Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 246 seconds","date":"2019-04-17T02:13:17.991Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T02:29:22.497Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 246 seconds","date":"2019-04-17T02:34:04.327Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T02:48:44.287Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 255 seconds","date":"2019-04-17T02:53:15.199Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T03:08:26.121Z","type":"join"}
{"nick":"dbevenius","reason":"Ping timeout: 246 seconds","date":"2019-04-17T03:12:47.981Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T03:22:54.253Z","type":"join"}
{"nick":"dbevenius","reason":"Remote host closed the connection","date":"2019-04-17T05:00:05.555Z","type":"quit"}
{"nick":"dbevenius","date":"2019-04-17T05:08:49.474Z","type":"join"}
{"nick":"dbevenius","reason":"Remote host closed the connection","date":"2019-04-17T05:12:04.653Z","type":"quit"}
{"nick":"Trott","message":"Something happened with the most recent coverage run such that sorting no longer works in the output. Results from two nights ago (https://coverage.nodejs.org/coverage-1e7823dd4ec1e0d0/index.html) is sortable by column (click on one). Most recent (https://coverage.nodejs.org/coverage-09cdc3782488d2e6/index.html) is not and a relevant-seeming error shows up in the console. Anyone know if anything changed recently with the","date":"2019-04-17T05:12:10.429Z","type":"message"}
{"nick":"Trott","message":"coverage job?","date":"2019-04-17T05:12:10.500Z","type":"message"}
{"nick":"dbevenius","date":"2019-04-17T05:12:16.108Z","type":"join"}
{"nick":"richardlau","message":"Trott: https://coverage.nodejs.org/coverage-09cdc3782488d2e6/sorter.js and https://coverage.nodejs.org/coverage-1e7823dd4ec1e0d0/sorter.js are different. I think it's https://github.com/istanbuljs/istanbuljs/pull/289 -- I'm guessing a new module was published (we ignore C8's package-lock.json when installing in our Makefile).","date":"2019-04-17T06:54:57.762Z","type":"message"}
{"nick":"richardlau","message":"Raised https://github.com/istanbuljs/istanbuljs/pull/369","date":"2019-04-17T07:35:17.234Z","type":"message"}
{"nick":"sxa_","date":"2019-04-17T09:37:54.430Z","type":"join"}
{"nick":"sxa","date":"2019-04-17T09:38:13.876Z","type":"join"}
{"nick":"Trott","message":"Thanks!","date":"2019-04-17T13:49:10.894Z","type":"message"}
{"nick":"Trott","message":"Is https://ci.nodejs.org/job/node-test-binary-windows-2/203/COMPILED_BY=vs2017,RUNNER=win2008r2-vs2017,RUN_SUBSET=0/testReport/(root)/test/parallel_test_child_process_fork_exec_path/ a problem with the test or a problem with the host? refack","date":"2019-04-17T13:50:13.578Z","type":"message"}
{"nick":"refack","message":"it's a problem with the test. Seems like `.tmp.2\\node-copy.exe` is still running, but it might be left over from a previous test, and hit here because it calls `tempdir.refresh()`","date":"2019-04-17T13:52:26.408Z","type":"message"}
{"nick":"Trott","message":"Two different node executables in `parallel` should not be able to share the same `.tmp` directory. At least in theory....","date":"2019-04-17T14:34:41.370Z","type":"message"}
{"nick":"refack","message":"I've been running that test locally, and it's actually a leftover of the same test file.","date":"2019-04-17T14:36:23.045Z","type":"message"}
{"nick":"refack","message":"is too tight https://www.irccloud.com/pastebin/SFQRsdBs/test.js","date":"2019-04-17T14:36:37.709Z","type":"message"}
{"nick":"Trott","message":"Yeah, so either Windows child exits before it actually, you know, exits...Or something else, but the cleanup is unnecessary. At a minimum, it should probably be wrapped in a try/catch. Maybe removed entirely. Depends on how big the leftover file will be.","date":"2019-04-17T14:37:43.858Z","type":"message"}
{"nick":"refack","message":"It's a node binary, ~20MB","date":"2019-04-17T14:40:02.938Z","type":"message"}
{"nick":"Trott","message":"Test usually passes but just fails once in a while for you locally?","date":"2019-04-17T14:41:06.024Z","type":"message"}
{"nick":"refack","message":"It doesn't fail on Windows 10, but does on 2008R2. It's a speed thing. The OS keeps the handle open just a tad bit longer","date":"2019-04-17T14:42:19.241Z","type":"message"}
{"nick":"Trott","message":"I guess the question is: Is that a bug? (I would say it is. But there also might not be a whole lot we can do about it?)","date":"2019-04-17T14:43:11.431Z","type":"message"}
{"nick":"refack","message":"2008R2 goes EOL on 2019-12-31","date":"2019-04-17T14:43:38.057Z","type":"message"}
{"nick":"refack","message":"It's one of those things that `gracefull-fs` was made for","date":"2019-04-17T14:44:03.860Z","type":"message"}
{"nick":"refack","message":"https://github.com/libuv/libuv/pull/2122","date":"2019-04-17T14:44:48.337Z","type":"message"}
{"nick":"Trott","message":"Well, if it's only failing very rarely and we won't need to worry about it after December, I guess I'm OK with ignoring it for now.... How are you feeling about it?","date":"2019-04-17T14:46:59.721Z","type":"message"}
{"nick":"refack","message":"https://ci.nodejs.org/job/node-stress-single-test/2201/nodes=win2008r2-vs2017/console","date":"2019-04-17T14:47:32.103Z","type":"message"}
{"nick":"dbevenius","reason":"Remote host closed the connection","date":"2019-04-17T14:47:46.151Z","type":"quit"}
{"nick":"refack","message":"strong enough to try and mask it https://github.com/nodejs/node/pull/27277","date":"2019-04-17T14:48:30.660Z","type":"message"}
{"nick":"dbevenius","date":"2019-04-17T15:33:03.523Z","type":"join"}
{"nick":"refack","message":"Meanwhile I found that `test/parallel/test-child-process-exec-timeout.js` is leaving a Zombie process behind...","date":"2019-04-17T15:47:45.913Z","type":"message"}
